.github/issue_template.md

```diff
@@ -40,8 +40,8 @@
     Thank you for contributing to npm! Please review this checklist
     before submitting your issue.
 
-    - Please check if there's a solution in the troubleshooting wiki:
-      https://github.com/npm/npm/blob/latest/TROUBLESHOOTING.md
+    - Please check if there's a solution in the troubleshooting posts on npm.community:
+      https://npm.community/c/support/troubleshooting
```

Comment:
You can actually remove this file entirely. Issues are not enabled, so no one will ever see this template.

Reply:
That's right! Updated.